util/Migrator/DbScripts/2024-03-06_00_ProviderAddGatewayColumns.sql

```diff
@@ -0,0 +1,186 @@
+-- Add 'Gateway' column to 'Provider' table.
```

Comment:
❓ Why three scripts vs. one? Thinking about transactional integrity of the migration.